app/views/collections/form.html.erb

```diff
@@ -13,28 +13,29 @@
 <% end %>
 <%= render Elements::HeaderComponent.new(level: :h1, variant: :h2, text: @collection_form.title || 'Untitled collection', classes: 'mb-3') %>
 <%= render Elements::AlertComponent.new(variant: :danger, value: I18n.t('collections.edit.messages.validation')) if @collection_form.errors.present? %>
-<%= render Elements::TabForm::TabListComponent.new(model: @collection_form, hidden_fields: %i[lock version]) do |component| %>
+<% form_id = dom_id(@collection_form, 'tabbed_form') %>
```

Comment:
I pondered whether we might expect this to be set as an instance variable in the controller instead of a local variable, but I don't think it matters too much?

Reply:
Yeah, my thought it that it made it a tad clearer to put it closer to where it was used at the expense of slightly more complicated erb.